sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CopyAuthorization.java

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.azure.core.util.serializer.SerializerAdapter;
+import com.azure.core.util.serializer.SerializerEncoding;
+
+import java.io.IOException;
+
+/**
+ * The CopyAuthorization model.
+ */
+@Fluent
+public final class CopyAuthorization {
+    private static final SerializerAdapter SERIALIZER = new JacksonAdapter();
+
+    CopyAuthorization() {
+        super();
+    }
+
+    /*
+     * Model identifier.
+     */
+    private String modelId;
+
+    /*
+     * Token claim used to authorize the request.
+     */
+    private String accessToken;
+
+    /*
+     * Resource Identifier.
+     */
+    private String resourceId;
+
+    /*
+     * Region of the resource.
+     */
+    private String region;
+
+    /*
+     * The time when the access token expires. The date is represented as the
+     * number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.
+     */
+    private long expiresOn;
+
+    /**
+     * Create a CopyAuthorization object
+     *
+     * @param modelId The model identifier
+     * @param accessToken The token used to authorize the request
+     * @param resourceId The resource identifier
+     * @param region The region of the resource
+     * @param expiresOn The expiry time of the token
+     */
+    public CopyAuthorization(final String modelId, final String accessToken, final String resourceId,
+        final String region, final long expiresOn) {
+        this.modelId = modelId;
+        this.accessToken = accessToken;
+        this.resourceId = resourceId;
+        this.region = region;
+        this.expiresOn = expiresOn;
+    }
+
+    /**
+     * Get the modelId property.
+     *
+     * @return the modelId value.
+     */
+    public String getModelId() {
+        return this.modelId;
+    }
+
+    /**
+     * Get the token claim used to authorize the request.
+     *
+     * @return the accessToken value.
+     */
+    public String getAccessToken() {
+        return this.accessToken;
+    }
+
+    /**
+     * Get the time when the access token expires. The date is represented as the
+     * number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.
+     *
+     * @return the expirationDateTimeTicks value.
+     */
+    public long getExpiresOn() {
+        return this.expiresOn;
+    }
+
+    /**
+     * Get the Azure Resource Id of the target Form Recognizer resource
+     * where the model will be copied to.
+     *
+     * @return the resourceId value.
+     */
+    public String getResourceId() {
+        return resourceId;
+    }
+
+    /**
+     * Get the location of the target Form Recognizer resource.
+     *
+     * @return the resourceRegion value.
+     */
```

Comment:
Semi-related to @kristapratico comment:

Semi-related to this: @maririos and @samvaity : I notice that you guys both have `region` as the name in `CopyAuthorization`, but `getCopyAuthorization` takes `resourceRegion` (java ex [here](https://github.com/Azure/azure-sdk-for-java/blob/fb8f100ddefc172e9acc16c04a98a2bad44b8c2e/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingAsyncClient.java#L430))

Should these be standardized?

Reply:
I think so too that we should standardize on this. I see that python calls its resource_region ([here](https://github.com/Azure/azure-sdk-for-python/blob/a8a90d298a2358d6443238d0a0fe3935a44401a8/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py#L258)). 
@maririos Do you think we can call it `resourceRegion` even on the Model?

Reply:
we decided to leave the method signatures constant across languages, (`resourceRegion`) but for the types in `CopyAuthorization` object, that only exists in Java and .NET, .NET team was more inclined to leave it as Region, as it is already understood that the information here is about a resource. 
Other point is that this property is not available for the public in .NET so the name could always change.
@samvaity could u remind me again, why is it that we don't hide this properties in Java?

Reply:
@maririos https://github.com/Azure/azure-sdk-for-java/pull/11403#discussion_r431995348
I will update this field to be called `resourceRegion` as in Java we are exposing it and will help maintain consistency.